package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/MeshImporter.cs

```diff
@@ -603,55 +676,120 @@ public class GeometrySubsets {
 #endif
     }
 
-    static void LoadPrimvars(Scene scene,
-                             Mesh unityMesh,
-                             string usdMeshPath,
-                             List<string> primvars,
-                             int[] faceVertexCounts,
-                             int[] faceVertexIndices) {
-      if (primvars == null || primvars.Count == 0) { return; }
+    private static Vector3[] SanitizePointsForUnity(Vector3[] vertices, int[] indices)
```

Comment:
Name game: this is not really "sanitizing" points, it's really expanding the indices (or unrolling them?).